.github/workflows/sitespeed.yml

```diff
@@ -39,6 +30,15 @@ jobs:
           responseCode: 200
           timeout: 200000
           interval: 500
+      - name: Login MAAS admin
+        run: |
+          for i in {1..5}; do
```

Comment:
Why is this set of instructions running in a loop?

Reply:
Will add a comment to explain